release.sh

```diff
@@ -13,6 +13,12 @@ increment_version() {
  echo "$v" | perl -pe s/$rgx.*$'/${1}'`printf %0${#val}s $(($val+1))`/
 }
 
+echo ""
+echo "The release script requires several external command line tools:"
+echo " - git"
+echo " - mvn"
+echo " - hub (https://hub.github.com/)"
```

Comment:
Can we use the official command line client instead? https://cli.github.com/

Reply:
Or is the benefit of using hub that we don’t have to log in again?

Reply:
Btw. We need to test that hub (or whatever we end up with) works as an early step in the script. So it doesn’t crash half way through.

Reply:
I wasn't aware there was another client, I thought `hub` _is_ the official client. Happy to consider alternatives if they're better.

Reply:
> Or is the benefit of using hub that we don’t have to log in again?

No and actually that brings up a point: you need to set this tool up first before it works.